xmtp_id/src/associations/hashes.rs

```diff
@@ -1,11 +1,15 @@
+use std::str::FromStr;
+
+use ethers::{types::H160, utils::to_checksum};
 use sha2::{Digest, Sha256};
 
 use super::AssociationError;
 
 /// Helper function to generate a SHA256 hash as a hex string.
 fn sha256_string(input: String) -> String {
+    let normalized_input = input.to_lowercase();
```

Comment:
If we want to do this here we should rename the function. Will be a footgun to change the string before hashing, since callers won't expect that.